client/src/components/ContestList/ContestList.tsx

```diff
@@ -0,0 +1,42 @@
+import Grid from '@material-ui/core/Grid';
+import List from '@material-ui/core/List';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+import Button from '@material-ui/core/Button';
+import useStyles from './useStyles'
+import LionTatoo from "../../Images/lionTatoo.png";
+
+
+export default function ContestList(): JSX.Element {
+    const classes = useStyles();
+
+    return (
+        <>
+        <List className={classes.list}>
```

Comment:
You can configure your code formatter to format on save so that you don't face spacing/indentation issues :)